dev/stats/get_important_pr_candidates.py

```diff
@@ -267,9 +279,12 @@ def score(self):
         # If the body contains over 2000 characters, the PR should matter 40% more.
         # If the body contains fewer than 1000 characters, the PR should matter 20% less.
         #
+        # Weight PRs with #protm tags more heavily:
+        # If there is at least one #protm tag, multiply the interaction score by the number of tags, up to 3.
+        interaction_score = min(self.interaction_score + 1, 3)
```

Comment:
Is this right ? We don't take into account the `self.protm_score`. Should this be: 
```
interaction_score *= min(self.protm + 1, 3)
```

Reply:
This looks good; made the change. Thanks!